README.md

```diff
@@ -4,13 +4,14 @@ Extremely fast utf8-only stream implementation to write to files and
 file descriptors.
 
 This implementation is partial, but support backpressure and `.pipe()` in is here.
-However, it is 20x faster than Node Core `fs.createWriteStream()`:
+However, it is 2-3x faster than Node Core `fs.createWriteStream()`:
```

Comment:
has the number dropped because Node's gotten faster?

Reply:
Yes! 🚀 

I also adjusted the benchmark to be more realistic